tests/wicked/before_test.pm

```diff
@@ -52,28 +52,22 @@ sub run {
         systemctl 'enable dhcpd.service';
         systemctl 'start dhcpd.service';
     }
-    if (check_var('WICKED', 'basic')) {
-        assert_script_run("rcwickedd restart");
-        unless (get_var('IS_WICKED_REF')) {
-            # Remove previous static config and leave dynamic one, only for SUT
-            assert_script_run('rm /etc/sysconfig/network/ifcfg-' . $ctx->iface());
-            $self->get_from_data("wicked/dynamic_address/ifcfg-eth0", '/etc/sysconfig/network/ifcfg-' . $ctx->iface());
-        }
+
+    if (check_var('WICKED', 'basic') && check_var('IS_WICKED_REF', '0')) {
+        # Remove previous static config and leave dynamic one, only for SUT
+        $self->get_from_data("wicked/dynamic_address/ifcfg-eth0", '/etc/sysconfig/network/ifcfg-' . $ctx->iface());
     }
     elsif (check_var('WICKED', 'advanced') || check_var('WICKED', 'startandstop')) {
-        assert_script_run("rcwickedd restart");
         record_info('INFO', 'Setup OpenVPN');
         zypper_call('--quiet in openvpn', timeout => 200);
         if (check_var('WICKED', 'advanced')) {
             record_info('INFO', 'Setup OVS');
-            zypper_call('--quiet in openvswitch', timeout => 200);
+            zypper_call('-q in openvswitch libteam-tools libteamdctl0 python-libteam', timeout => 200);
             assert_script_run("systemctl start openvswitch");
         }
     }
-    elsif (check_var('WICKED', '2nics')) {
-        assert_script_run('rm /etc/sysconfig/network/ifcfg-' . $ctx->iface());
-        assert_script_run("rcwickedd restart");
-    }
+    assert_script_run('rm /etc/sysconfig/network/ifcfg-' . $ctx->iface());
```

Comment:
Does this  mean, that we now avoid having network at the beginning of each test?
And pre-run and post-run use `setup_static_network()` to do upload_logs and so on.

Reply:
yes , I think  it is right approach . we already have a few scenarios where fact that eth0 has IP before test starts fail test case ... So I really waiting for offline upload/download feature to be done so we can do all thing in a right way :)

Reply:
Ok I like the idea of getting a untouched network in each test.
What do you think about just doing `rm /etc/sysconfig/network/*` ?

Reply:
We need to keep `/etc/sysconfig/network/config` but everything else...